draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx

```diff
@@ -105,7 +105,12 @@ export const VideoPlayer = ({
          */
       }
     }
-  }, [prefersReducedMotion])
+    /**
+     * Chrome seems to have an issue with changes to autoplay after the video
+     * has been mounted. If there is a change in autoplay we need to force the
+     * play() method to be called.
+     */
+  }, [prefersReducedMotion, autoplay])
```

Comment:
I opened an animated scene illustration in Chrome (https://dev.cultureamp.design/ally/safari-video/storybook/?path=/story/components-illustration-scene--brand-moments) to see if it autoplayed, but it works intermittently. Are you getting the same behaviour? I.e. it autoplays on some page loads but not on others.

Looking at this component, and how there are `useEffect` hooks which depend on `videoRef`, my gut says `videoRef` should probably be from a `useState` rather than `useRef`?
Reason being, the latter (React refs) won't cause the consecutive `useEffect`s to be re-run because they don't re-render the component when they change. It might be working intermittently because the video element is sometimes mounted in time, so `videoRef` is not null during the effects.
Honestly just hunch could be wrong!

Reply:
I had a quick crack, setting the `muted` prop on the video element did the trick for me. I was seeing an error like `play() failed because the user didn't interact with the document first` when looking closer.

![image](https://user-images.githubusercontent.com/7651503/128787682-94047c92-0040-453a-a5b3-e029d03de642.png)

Reply:
Nice - thanks for flagging this, I completely missed the flakiness. You're right, I should probably also be setting muted on the `video` itself. Let me make those changes :loading:

Reply:
I've just set the `muted` prop on the `video` and I can't see any intermittent issues anymore.

Reply:
likewise it's always playing now!